### PR TITLE
[AdminBundle] add styling to range slider

### PR DESCRIPTION
**Pull request by LiesbethVanaerschot**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | Range slider not styled correctly + add value indication with js #473

### CODE REVIEW

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            initRangePercentage(el);
+
+        }).trigger('change');
+    }
+
+    //Initialize
+    initRangePercentage = function(el) {
+
+        // Measure width of range input
+        /*wdth = el.width();
```
Comment (FVKVN) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js]: Delete commented lines if you don't use them

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+kunstmaanbundles.rangeslider = (function(window, undefined) {
+
+    var init, reInit, initRangePercentage;
+    var el, newPoint, newPlace, offset;
+
+    init = reInit = function() {
+        $(".range").find('input').change(function() {
+
+            el = $(this);
```
Comment (FVKVN) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js]: prefix with $ if it's a jquery object

File: src/Kunstmaan/AdminBundle/Resources/ui/scss/default-theme/components/forms/_form-control.scss [hunk 3]
```diff
@@ -77,3 +77,75 @@ input[type=file].form-control {
...
+    -webkit-appearance: none;
+    background-color: $accent-color;
+    width: 15px;
+    height: 15px;
+    border-radius: 10px;
+    cursor: pointer;
+}
+
+input[type=range]::-ms-thumb {
+    -webkit-appearance: none;
```
Comment (FVKVN) [inline src/Kunstmaan/AdminBundle/Resources/ui/scss/default-theme/components/forms/_form-control.scss]: order

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        $('.range--value')
+            .text(el.val());
+    };
+
+    return {
+        init: init,
+        reInit: reInit
+    };
+
+}(window));
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js]: }()); -> })();

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
+var kunstmaanbundles = kunstmaanbundles || {};
+
+kunstmaanbundles.rangeslider = (function(window, undefined) {
+
+    var init, reInit, initRangePercentage;
+    var el, newPoint, newPlace, offset;
+
+    init = reInit = function() {
+        $(".range").find('input').change(function() {
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js]: Re-init will attach double eventhandlers. Does this cause issues?

File: src/Kunstmaan/AdminBundle/Resources/ui/scss/default-theme/components/forms/_form-control.scss [hunk 6]
```diff
@@ -77,3 +77,84 @@ input[type=file].form-control {
         transform: none;
     }
 }
+
+/* Range slider
+   ===========================================================================*/
+
+.range {
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/scss/default-theme/components/forms/_form-control.scss]: Remove unused selectors

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js (hunk 5 above)
Comment (FVKVN) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js]: Check if the element isn't already initialised? (cfr. the characters-left.js file)

**Pull request merged**